src/main/java/pages/TodaySDealsPage.java

```diff
@@ -57,14 +62,21 @@ public boolean anyCheckboxIsSelected() {
     }
 
     public void waitForTwoFilterItemsLoaded() {
-        List<WebElement> listOfFilterItems = driver.
-                findElements(By.xpath("//div[contains(@class, 'DealGridItem-module__dealItemContent')]"));
-        new WebDriverWait(driver, Duration.ofSeconds(20)).
+        WebDriverWait wait = new WebDriverWait(driver, ofSeconds(15));
+        wait.until((ExpectedCondition<Boolean>) driver -> {
+            if (listOfFilterItems.size() > 0)
+                return true;
+            else
+                return false;
+        });
```

Comment:
Do you know what you've written? What does it mean this "->" sign and what it is for?
Delete and write a code, that you understand how works.